server.js

```diff
@@ -2,21 +2,39 @@ const express = require('express')
 const app = express()
 const fs = require('fs')
 const path = require('path')
+const serialize = require('serialize-javascript')
+const { createBundleRenderer } = require('vue-server-renderer')
+let renderer
 
 const indexHTML = (() => {
   return fs.readFileSync(path.resolve(__dirname, './index.html'), 'utf-8')
 })()
 
 app.use('/dist', express.static(path.resolve(__dirname, './dist')))
 
-require('./build/dev-server')(app)
+require('./build/dev-server')(app, bundle => {
```

Comment:
I think you basically moved server.js in the "src" folder. In the package.json it runs "node server", so it would run this file not the "src/server.js" file!